src/modules/chartEditor/saga/restoreSavedQuery.test.ts

```diff
@@ -0,0 +1,99 @@
+import { Query } from '@keen.io/query';
+import sagaHelper from 'redux-saga-testing';
+import { getContext, put, select } from 'redux-saga/effects';
+import { PUBSUB } from '../../../constants';
+import { chartEditorSelectors } from '../selectors';
+import { chartEditorActions } from '../index';
+import {
+  SET_CHART_SETTINGS,
+  SET_QUERY_EVENT,
+} from '@keen.io/query-creator/dist';
```

Comment:
is dist required here?

Reply:
please organize your imports

for example
- external sources
- @keen.io
- utils
- constants